docs/data/base/components/badge/badge.md

```diff
@@ -47,6 +153,6 @@ The default is 99.
 ## Accessibility
 
 Screen readers may not provide users with enough information about a badge's contents.
-To make your `BadgeUnstyled` accessible, you must provide a full description with `aria-label`:
+To make your badge accessible, you must provide a full description with `aria-label`, as shown in the demo below:
 
-{{"demo": "AccessibleBadges.js", "defaultCodeOpen": false}}
+{{"demo": "AccessibleBadges.js"}}
```

Comment:
I think that we could keep the preview disabled. It doesn't explain how the feature works, so I feel noise for developers

```suggestion
{{"demo": "AccessibleBadges.js", "defaultCodeOpen": false}}
```